### DISCUSSION TITLE
Syntax errors in Clang while using android-ndk-r25c-linux.zip

### DISCUSSION THREAD

**Discussion by remul19** [android/ndk#1901] [Q&A]
Hi All,

I downloaded the latest Linux android-ndk from https://developer.android.com/ndk/downloads .

I added the folder path in bashrc and tried to cross compile a code for android. But got below error messages:
 `/home/myTest/android_ndk/android-ndk-r25c/toolchains/llvm/prebuilt/linux-x86_64/bin/clang: 1: Syntax error: word unexpected (expecting ")")`

```
-- Check for working C compiler: /home/myTest/android_ndk/android-ndk-r25c/toolchains/llvm/prebuilt/linux-x86_64/bin/clang -- broken
CMake Error at /usr/share/cmake-3.16/Modules/CMakeTestCCompiler.cmake:60 (message):
  The C compiler

    "/home/myTest/android_ndk/android-ndk-r25c/toolchains/llvm/prebuilt/linux-x86_64/bin/clang"

  is not able to compile a simple test program.
```
Could anyone please help me to resolve this issue?

**Comment by bingo787** [comment]
I meet too！ r27
